script/setup

```diff
@@ -15,12 +15,25 @@ fi
 script/bootstrap
 
 if psql -lqt | cut -d \| -f 1 | grep -w challenges_dev; then
-  echo "Database already exists"
-else
-  echo "===> Setting up DB..."
-  createdb challenges_dev
+  echo "Development database already exists, dropping"
+  dropdb challenges_dev
 fi
-script/mock-data
 
+echo "===> Setting up development database..."
+createdb challenges_dev
+
+echo "==> Migrating development"
+./node_modules/db-migrate/bin/db-migrate up --config config/database.json -m db/migrations
```

Comment:
npm helpfully symlinks all bins (as long as they're declared as such in the module's `package.json`) to `./node_modules/.bin/bin_name`. Typically this path is preferred to manually referencing the path inside the module, as long as the module declares them.